### PR TITLE
Review chain helpers - Closes #3035

### LINKED ISSUE #3035: Review chain/helpers/ content
### Expected behavior

Only real utils should be placed in helpers folder.

### Actual behavior

There are too many files in the folder chain/helpers/. Most of them are obsoleted, not real utils or simply not well placed.  

### Which version(s) does this affect? (Environment, OS, etc...)

`1.6`

### PR DESCRIPTION
**Pull request by diego-G**
### Review checklist

* The PR resolves #3035
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: framework/src/modules/chain/helpers/dapp_types.js [hunk 1]
```diff
@@ -1,30 +0,0 @@
...
- *
- * @namespace dapp_types
- * @memberof helpers
- * @see Parent: {@link helpers}
- * @property {number} DAPP
- * @property {number} FILE
- * @todo Add description for the namespace and the properties
- */
-module.exports = {
-	DAPP: 0,
```
Comment (shuse2) [inline framework/src/modules/chain/helpers/dapp_types.js]: this one was not used?

Comment (diego-G) [reply framework/src/modules/chain/helpers/dapp_types.js]: Apparently not.

**Comment by limiaspasdaniel** [comment]
@diego-G, regarding @MaciejBaj comment, let me know if you do something with `helpers/transaction_types.js`, maybe it solves code duplication on HTTP API too.

File: framework/src/controller/schema/constants.js [hunk 2]
```diff
@@ -264,4 +268,54 @@ module.exports = {
...
+			'VOTE',
+			'MULTI',
+			'DAPP',
+			'IN_TRANSFER',
+			'OUT_TRANSFER',
+		],
+		properties: {
+			SEND: {
+				type: 'integer',
+				min: 0,
```
Comment (MaciejBaj) [inline framework/src/controller/schema/constants.js]: min doesn't make sense - can it be equal? If not then both min and max

Comment (MaciejBaj) [reply framework/src/controller/schema/constants.js]: enum: [0],

**Comment by shuse2** [comment]
Also, I realised that `slot.js` is left behind in the helper, can we move it to logic or to appropriate round file?

**Comment by diego-G** [comment]
@shuse2 `helper/slots.js` is used in many places. There are several logics, several submodules and two api controllers that make use of that util. I reckon it should be kept as a helper.

**Comment by shuse2** [comment]
As discussed, please move the slot file to the `logic` folder for now

File: framework/src/modules/chain/logic/vote.js [hunk 3]
```diff
@@ -49,20 +47,25 @@ let self;
...
 			account,
+			transaction,
 		};
 		__scope.schema = schema;
 		// TODO: Add modules to contructor argument and assign delegates module to __scope.modules.delegates
 	}
 
 	/**
-	 * Calls Diff.reverse to change asset.votes signs and merges account to
+	 * Calls transaction.reverse to change asset.votes signs and merges account to
```
Comment (limiaspasdaniel) [inline framework/src/modules/chain/logic/vote.js]: 👌🏼

**Comment by diego-G** [comment]
@shuse2 due to several hurdles and the current PR's size, I decide to open an independent issue to tackle your suggestion about helper slots. https://github.com/LiskHQ/lisk/issues/3070

**Pull request merged**